workflow/rocoto_funcs/base.py

```diff
@@ -107,27 +136,34 @@ def __init__(self,task_id,cycledefs,maxtries,dcTaskRes,realtime=False,deadline="
     self.dcTaskEnv=dcTaskEnv
     self.dependencies=dependencies
 
+  def wflow_task_divider(self,xmlFile):
+    text=f'\n<!--\n'
+    text=text+f'************************************************************************************\n'
+    text=text+f'************************************************************************************'
+    text=text+f'\n-->\n'
+    xmlFile.write(text)
+
   def wflow_task_begin(self,xmlFile):
     text=f'\n<task name="{self.task_id}" cycledefs="{self.cycledefs}" maxtries="{self.maxtries}">\n'
     xmlFile.write(text)
 
   def wflow_task_part1(self,xmlFile): #write out part1 which excludes dependencies
-    text=f'  <command>{self.dcTaskRes["command"]}</command>\n'
+    text=f'  <command>{self.dcTaskRes["command"]} &HOMErrfs;</command>\n'
     text=text+f'  <join><cyclestr>{self.dcTaskRes["join"]}</cyclestr></join>\n'
     text=text+f'\n  <jobname><cyclestr>{self.dcTaskRes["jobname"]}</cyclestr></jobname>\n'
-    text=text+f'  <account>{self.dcTaskRes["account"]}</account>\n'
-    text=text+f'  <queue>{self.dcTaskRes["queue"]}</queue>\n'
-    text=text+f'  <partition>{self.dcTaskRes["partition"]}</partition>\n'
+    text=text+f'  <account>&ACCOUNT;</account>\n'
```

Comment:
This will reduce our flexibility of using different ACCOUNT/QUEUE/PARTITION/RESERVATION for different tasks. I think it is a step back.

Reply:
Those are not changed for difference tasks. We may need another group of those variable for different task. But we do not need to set those different for each task.